build/lender/Lender.sol

```diff
@@ -435,6 +453,10 @@ contract Lender {
         // Determine the amount lent after fees
         uint256 lent = a - fee;
 
+        // Deposit into aave
+        // TODO: Get a referral code and use it to deposit
+        IAave(aave).deposit(u, lent, address(this), 0);
+
         // Swap on the APWine Pool using the provided market and params
         uint256 returned = IAPWineRouter(pool).swapExactAmountIn(i, 1, lent, 0, r, address(this));
```

Comment:
those hard coded values look mysterious `0` in 458 and `1` in 461.

it's prob something we are stuck with tho, not ganna make any sort of demands on this, just noting it for posterity

Reply:
Yes, we are stuck with this one in particular. It's a referral code that used to be part of Aave's incentive system. I believe it is now an unused parameter (i.e. we could send anything in for that parameter). That said, the docs do recommend going with 0 for this parameter if the user doesn't have a referral code.

Reply:
S.
M.
H.

WEB3 BRO